dataset-cellsystems-2021/fish/gene_pairs.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "StructureId/Name": "MYH6-MYH7",
+        "ProteinId/DisplayName": "MYH6-MYH7",
+        "CellLineId/Name": "MYH6-MYH7",
+        "GeneId/Name": "MYH6-MYH7",
+        "GeneId/FullName": "MYH6-MYH7",
+        "ProteinId/Name": "MYH6-MYH7",
+        "Channels": "561-638"
```

Comment:
`Channels` is not a part of this data structure, right?

Reply:
right